03_Modules/EVDriver/src/module/module.ts

```diff
@@ -392,6 +402,15 @@ export class EVDriverModule extends AbstractModule {
                   }
                 }
               }
+
+              for (const authorizer of this._authorizers) {
+                if (response.idTokenInfo.status !== AuthorizationStatusEnumType.Accepted) {
+                  break;
+                }
+                const result: Partial<IdTokenType> = await authorizer.authorize(authorization, context);
+                Object.assign(response.idTokenInfo, result);
+              }
```

Comment:
This is just personal preference.

```suggestion
if (response.idTokenInfo.status === AuthorizationStatusEnumType.Accepted) {  
  for (const authorizer of this._authorizers) {
    const result: Partial<IdTokenType> = await authorizer.authorize(authorization, context);
    Object.assign(response.idTokenInfo, result);
  }
}
```

Reply:
This doesn't work; if there's a rejection in the for loop it no longer breaks.